.github/workflows/build-library.yml

```diff
@@ -71,14 +71,8 @@ jobs:
         with:
           node-version: 13.3.0
           registry-url: https://registry.npmjs.org/
-      - run: npm install
-      - name: Publish if version has been updated
-        uses: pascalgn/npm-publish-action@1.1.0
-        with:
-          tag_name: "v%s"
-          tag_message: "v%s"
-          commit_pattern: "^Release (\\S+)"
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          NPM_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+      - run: npm ci
+      - run: npm build
```

Comment:
Do we need to build besides installing with `npm ci`?

Reply:
Yes, we do. Install, just installs the dependencies. We need to run the build actions in the build script.

Reply:
Yes, we need to generate the `dist` folder for publishing.

Reply:
Although we could download the artifact from the build job instead and save some time.

Reply:
Good idea, then we can even skip the npm install step.